Altered Mission Statement

syberphunk opened the pull request:
Made adjustments to direct people to relevant information for each
wording that may require more explanation. Also expanded shortened form
of words that may be open to ambiguity unnecessarily (spaces as opposed
to referring to Hackspace where it is the UK Hackspace Foundation of
Hackspace and not to be confused with Hackerspace or other definitions
of the word).  Also expanded wording to refer to ourselves as the UK
Hackspace Foundation in fulll as opposed to shortening to 'Foundation'
for clarity of which is being referred to. Adjusted the flow of the
paragraphs to intend to read as an introduction to the foundation and
terms, an explanation of action and an outro of future development,
wording was also adjusted to attempt to not purposefully exclude those
outside of the Hackspace definition and that this is open to alteration.

Also removed references to other files where those other files did not
elaborate on the word or wording linked from.

syberphunk committed:
Altered Mission Statement

Made adjustments to direct people to relevant information for each
wording that may require more explanation. Also expanded shortened form
of words that may be open to ambiguity unnecessarily (spaces as opposed
to referring to Hackspace where it is the UK Hackspace Foundation of
Hackspace and not to be confused with Hackerspace or other definitions
of the word).  Also expanded wording to refer to ourselves as the UK
Hackspace Foundation in fulll as opposed to shortening to 'Foundation'
for clarity of which is being referred to. Adjusted the flow of the
paragraphs to intend to read as an introduction to the foundation and
terms, an explanation of action and an outro of future development,
wording was also adjusted to attempt to not purposefully exclude those
outside of the Hackspace definition and that this is open to alteration.

Also removed references to other files where those other files did not
elaborate on the word or wording linked from.

idnorton reviewed (changes requested):
@syberphunk I like it, could you please update your branch to reflect the outcome of the regionalisation discussion on the forum?

https://forum.hackspace.org.uk/t/name-and-geographical-remit/208

I believe we're punting for "UK, Republic of Ireland, or Crown Dependencies" as @theorbtwo has added in his PR over at UKHackspaceFoundation/foundation#47

Happy to thumb it once changed, nice rewording work, thankyou for your contribution :)

syberphunk committed:
Locale references removed

Removed references to specifically UK locale and also clarified wording
to prevent ambiguity.

syberphunk commented:
Better @idnorton ? I figured it works by merely removing the majority of the localisation. Could readily add appropriate locale back in if needed, I believe these last commits should suit.

syberphunk committed:
Scope

Added in statement of location scope to first sentence.

idnorton reviewed (approved):
Awesomesauce!

Looks good to me, merging up.
